app/components/auth/dummy/ui/DummyLoginUI.js

```diff
@@ -0,0 +1,46 @@
+import { Button } from "react-bootstrap";
+import { NoUserAvatar } from "../../NoUserAvatar";
+import { dummyUserLogin } from "../lib/function";
+
+export default function DummyLoginUI({ user, setUser }) {
+  const login = () => {
+    const dummy_user = dummyUserLogin();
+    setUser(dummy_user);
+    sessionStorage.setItem("dummy_user", JSON.stringify(dummy_user));
+  }
+  const logout = () => {
+    setUser({});
+    sessionStorage.removeItem("dummy_user");
+  }
```

Comment:
Move them to `DummyLoginButton` and just pass the `user` state. Pass these as `handleLogin` and `handleLogout`.